### PR TITLE
created claim UI for kpi options

### PR DESCRIPTION
**Pull request by TotoFullStackDev**
##### Description
if you wanna test it, please set this custom RPC into your metamask wallet
url -> https://rpc.tenderly.co/fork/448f83e7-c39b-4e5d-8155-4bdafefb9b09
chainID -> 1
name -> the one you wish
all the rest empty

##### Checklist

- [ ] Linter status: 100% pass
- [ ] Changes don't break existing behavior
- [ ] Test coverage hasn't decreased

### CODE REVIEW

**Review by AdrianAddesso — CHANGES_REQUESTED** [review]
1. Align lock lenght to the left
image

2. Copy Changes
 a. Change **Total distributed rewards** for **Total Distributed KPI Options**
 b. Keep **Your Claimable KPI Options** as is
 c. Change **Your Estimated UMA KPI Options** for **Your Estimated Payout**

**Review by AdrianAddesso — APPROVED** [review]
All good. Approved

**Review by dimarconicola — APPROVED** [review]
Tested with custom network
- data shown are correct
- claim triggers metamask
- tx is casted
- kDOUGH is airdropped to my wallet
- my claimable balance goes to 0

**Comment by dimarconicola** [comment]
@AdrianAddesso should we review the content before merging this PR?

**Review by AdrianAddesso — CHANGES_REQUESTED** [review]
1. Copy change: **Use your kDOUGH to redeem the DOUGH payout** for **Use your kDOUGH to redeem the veDOUGH payout**
2. please adjust the padding of the numbered steps to 24px so that the distance between the number and the text is closer to what the design shows (_leave the Mobile version as is, otherwise there will be overlapping_)
image

3. Join our discord box should link to http://discord.link/PieDAO

4. The share button should have the following functionality:
    a) User clicks on the button 'Share'-> URL is copied in the clipboard -> button text changes to 'Link Copied!'
   b) The URL to be copied is /#/dough-staking-campaign
image

File: src/notifications.js [hunk 1]
```diff
@@ -4,9 +4,40 @@ import { isAddress, isTransactionHash } from '@pie-dao/utils';
...
+    block.transactions.forEach(async(transaction) => {
+      // we search for a match in the emitersArray...
+      let tnx = emittersArray.find(el => el.hash == transaction);
+      // if a transaction was stored into our emitersArray...
+      if(tnx) {
+        // we fetch the transaction receipt, and check the status...
+        let tnxReceipt = await provider.getTransactionReceipt(transaction);
+        // if status = 1 it means success, then we can trigger the event for it...
+        if(tnxReceipt.status == 1) {
+          tnx.emitter.emit('tnxConfirmed');
```
Comment (Alexintosh) [inline src/notifications.js]: wrong event name: tnxConfirmed

**Review by AdrianAddesso — CHANGES_REQUESTED** [review]
Sorry, just found an issue on the mobile version.
When user opens the nav menu, the progress bar milestones text overlaps with the menu text
image

**Review by AdrianAddesso — APPROVED** [review]
Changes approved

**Review by dimarconicola — APPROVED** [review]
+1

**Pull request merged**